test/splitter.js

```diff
@@ -0,0 +1,160 @@
+const truffleAssert = require('truffle-assertions');
+const Splitter = artifacts.require("./Splitter.sol");
+
+contract('Splitter', async accounts => {
+
+    let gasCost = async receipt => {
+        const gasUsed = receipt.receipt.gasUsed;
+        const tx = await web3.eth.getTransaction(receipt.tx);
+        const gasPrice = tx.gasPrice;
+
+        return web3.utils.toBN(gasUsed * gasPrice);
+    };
+
+    let checkEventNotEmitted = async eventName => {
```

Comment:
That's strange, you are checking that a specific event is not emitted. I am more used to testing that a specific event is emitted, or that no events are emitted.

Also, `const`.

Reply:
I wasn't aware that the emit type was optional. I have changed. Thank you.